test/app/modes/C6H6_ir/staticPol.out

```diff
@@ -0,0 +1,36 @@
+ -0.10130259E-11  0.73266928E+00  0.23399947E+01  0.73266825E+00  0.78159701E-09 -0.32314216E-04  0.23399914E+01  0.15493356E-03  0.28421709E-09
```

Comment:
We should rename this file to the name, which DFTB+ generates (`bornderiv.out`) by default.

Reply:
done